students/elmar_m/lesson07/html_render.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+'''
+file: html_render.py
+elmar_m / 22e88@mailbox.org
+Lesson07: HTML renderer
+'''
+
+class Element:
+    tag = 'element'
+    props = []
+    propstring = ''
+
+    def __init__(self, content=None, **kwargs):       
+        self.content = [content] if content else []  
+        self.attrs = kwargs
+        self.tagopen = self.tag
+
+        if self.attrs:
+            for key, value in self.attrs.items():
+                self.props.append('{}="{}"'.format(key,value))
+            self.propstring = ' '.join(self.props)
+            self.tagopen = '{} {}'.format(self.tag, self.propstring)
```

Comment:
I would move this logic out of `__init__` and into a separate method that returns the `tagopen` value. Then `prop` and `propstrings` can be variables defined within the scope of that method, rather than class attributes